src/util/SchemaUtil.cpp

```diff
@@ -290,6 +290,47 @@ StatusOr<DataSet> SchemaUtil::toShowCreateSchema(bool isTag,
     return dataSet;
 }
 
+StatusOr<DataSet> SchemaUtil::toDescIndex(const meta::cpp2::IndexItem &indexItem) {
+    DataSet dataSet({"Field", "Type"});
+    for (auto &col : indexItem.get_fields()) {
+        Row row;
+        row.values.emplace_back(Value(col.get_name()));
+        row.values.emplace_back(typeToString(col));
+        dataSet.emplace_back(std::move(row));
+    }
+    return dataSet;
+}
+
+StatusOr<DataSet> SchemaUtil::toShowCreateIndex(bool isTagIndex,
```

Comment:
why don't move to `IndexUtil` ?

Reply:
Sorry, I don't notice IndexUtil before. I have fixed it just now.